cmd/gcp-controller-manager/app/node_annotater.go

```diff
@@ -60,6 +62,21 @@ func newNodeAnnotator(client clientset.Interface, nodeInformer coreinformers.Nod
 	}
 	gce := compute.NewInstancesService(cs)
 
+	ownedKubeLabels := []string{
```

Comment:
Add a comment explaining what these are. I imagine this list will need to be maintained somehow.

Reply:
Is it possible to also read these owned labels from metadata? Was thinking it may be better to have a centralized place (GKE) to defining/modifying these. Or the coordination between GKE control plane and gcp-controller-manager might be a bit tricky.

Reply:
@mikedanese I'd still like a comment here with explanation. A TODO is fine too.

Reply:
Another thought, would it be helpful to make these overridable via a command line flag?